### PR TITLE
MBS-10046: Convert /admin/banner/edit to React

### PR DESCRIPTION
**Pull request by mwiencek**

### CODE REVIEW

File: root/admin/EditBanner.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+type Props = {|
+  +form: FormT<{|
+    +message: FieldT<string>,
+  |}>,
+|};
+
+const EditBanner = ({form}: Props) => (
+  <Layout fullWidth title={l('Edit Banner Message')}>
+    <div id="content">
+      <h1>{l('Edit Banner Message')}</h1>
```
Comment (reosarevok) [inline root/admin/EditBanner.js]: Very minor, but I'd much rather this was "Edit banner message" and we started moving towards consisting mostly-lowercase headers

Comment (yvanzo) [reply root/admin/EditBanner.js]: Would you be kind enough to write a small guide for future contributors? :pray: 
Should it be part of the design system or something?

Comment (mwiencek) [reply root/admin/EditBanner.js]: Fixed in cf065e6b92b7f799120e06f822f2d16b4c9c38a8

Comment (reosarevok) [reply root/admin/EditBanner.js]: I'm not sure how the design system should be :) But I'm happy to make my pet peeves official, sure :D Just let me know how.

**Comment by mwiencek** [comment]
I cherry-picked your commit and amended it with `required={required}` forwarded onto the textarea.

**Pull request merged**